workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/RadioCardList/components/RadioCard.tsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ClayRadio} from '@clayui/form';
+import ClaySticker from '@clayui/sticker';
+import classNames from 'classnames';
+
+import './RadioCard.scss';
+import emptyPictureIcon from '../../../assets/icons/avatar.svg';
+
+interface RadioCardProps<T> {
+	activeRadio: RadioOption<T> | undefined;
+	description?: string;
+	imageURL?: string;
+	index: number;
+	leftRadio?: boolean;
+	selectRadio: () => void;
+	showImage?: boolean;
+	title: string;
+}
+
+const NewRadioCard = <T extends unknown>({
+	activeRadio,
+	description,
+	imageURL,
+	index,
+	leftRadio,
+	selectRadio,
+	showImage,
+	title,
+}: RadioCardProps<T>) => {
+	return (
+		<div
+			className={classNames(
+				'align-items-center d-flex justify-content-between form-control mb-5 cursor-pointer py-4 px-0',
+				{
+					radioSelected: activeRadio?.index === index,
```

Comment:
Please, change this.
```suggestion
					"radio-selected": activeRadio?.index === index,
```